server/athenian/api/openapi/openapi.yaml

```diff
@@ -5079,3 +5155,105 @@ components:
           - $ref: '#/components/schemas/DeveloperSet'
           - description: Release publishers.
       type: object
+    JIRAMetricID:
+      description: |-
+        Currently supported JIRA activity metrics.
+        * `jira-bug-open` - number of bug-like issues active by the end of the time interval.
+        * `jira-bug-resolved` - number of bug-like issues closed in the given time range.
+        * `jira-bug-raised` - number of bug-like issues reported in the given time range.
+      enum:
+      - jira-bug-open
+      - jira-bug-resolved
+      - jira-bug-raised
+      example: jira-bug-resolved
+      type: string
+    JIRAMetricsRequest:
+      additionalProperties: false
+      description: Request body of `/metrics/jira`.
+      example:
+        account: 1
+        date_from: 2020-10-01
+        date_to: 2020-10-30
+        priorities:
+        - High
+        metrics:
+        - jira-bug-open
+        - jira-bug-resolved
+        - jira-bug-raised
+        granularities:
+        - all
+      properties:
+        account:
+          description: Session account ID.
+          type: integer
+        date_from:
+          description: Issues must be resolved after this date.
+          format: date
+          type: string
+        date_to:
+          description: Issues must be created before this date.
+          format: date
+          type: string
+        timezone:
+          $ref: '#/components/schemas/TimeZone'
+        priorities:
+          description: Selected issue priorities.
+          items:
+            type: string
+          type: array
+        assignees:
+          description: Selected issue assignee users.
+          items:
+            $ref: '#/components/schemas/DeveloperID'
+          type: array
+        reporters:
+          description: Selected issue reporter users.
+          items:
+            $ref: '#/components/schemas/DeveloperID'
+          type: array
+        commenters:
+          description: Selected issue commenter users.
+          items:
+            $ref: '#/components/schemas/DeveloperID'
+          type: array
+        stakeholders:
+          description: Selected issue stakeholder users.
+          items:
+            $ref: '#/components/schemas/DeveloperID'
+          type: array
+        metrics:
+          description: List of measured metrics.
+          items:
+            $ref: '#/components/schemas/JIRAMetricID'
+          type: array
+        granularities:
+          description: Splits of the specified time range `[date_from, date_to)`.
+          items:
+            $ref: '#/components/schemas/Granularity'
+          type: array
```

Comment:
Note: there are no `quantiles` here. If you think we are going to need them, we'd rather add them sooner than later. @warenlg

Reply:
We don't need quantiles for bug counts of course, but we do need them for MTTRs which are the next Jira metric to add. So I'd say yes, let's add quantiles now.